### PR TITLE
Add pranavarora1895

### PR DESCRIPTION
**Pull request by pranavarora1895**
Kindly Add my profile to the main project.
Account - [pranavarora1895](https://github.com/pranavarora1895)
Name - Pranav Arora

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1587"></a>

### CODE REVIEW

File: public/data/pranavarora1895.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      "url": "https://twitter.com/PranavArora95",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Facebook",
+      "url": "https://www.facebook.com/cosmicpranav007/",
+      "icon": "facebook"
+    },
+    {
+      "name": "Follow me on Linkedin",
```
Comment (kumarsonsoff3) [inline public/data/pranavarora1895.json:23]: What do you think on capitalizing `i` here!?

```suggestion
      "name": "Follow me on LinkedIn",
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inelegant

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
- Please Revert the changes from the `package-lock.json` file.
- Also, make the following changes in your file. 

Let us know if you need any help with this!


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant, Not maintainable

**Comment by kumarsonsoff3** [comment]
@pranavarora1895 Changes in `package-lock.json` file are not required! So please revert the changes from `package-lock.json` file! Let us know if you need any help! 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,
The `package-lock.json` file is still in the PR. If you need any help, feel to ping us.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by pranavarora1895** [comment]
> Hey, The `package-lock.json` file is still in the PR. If you need any help, feel to ping us.
> 
> __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** 🛠️ This code needs a fix

Hi. I am not able to revert the changes and put it back to this PR. Kindly let me know how to do it. Thanks.

**Comment by Panquesito7** [comment]
> > Hey, The `package-lock.json` file is still in the PR. If you need any help, feel to ping us.
> > __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  🛠️ This code needs a fix
> 
> Hi. I am not able to revert the changes and put it back to this PR. Kindly let me know how to do it. Thanks.

Just copy the contents from the [original `package-lock.json`](https://raw.githubusercontent.com/EddieHubCommunity/LinkFree/main/package-lock.json), paste them into your `package-lock.json` branch, commit, then push. You might have to revert the changes first if you're using a service such as Gitpod or running NPM each time you start the editor. Let us know if you need any help. 🙂

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Hey @pranavarora1895, Just remove the `empty endline` from the `package-lock.json` file! 

### ⚠ Remember: 
If you're using a service such as Gitpod or running `NPM` each time you start the editor, then you have to follow what @Panquesito7 said! 😄
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by kumarsonsoff3** [comment]
Do you need any help with this @pranavarora1895 !?

**Comment by EmmaDawsonDev** [comment]
Since there is no answer on this for a while I will close it. Please feel free to make a new pull request adding your portfolio without the package-lock file.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Tags:** Not reusable

**Pull request closed without merging**